src/web3/web3.service.ts

```diff
@@ -0,0 +1,62 @@
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import * as ContractInstance from '../contracts/Example.json';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const Web3 = require('web3');
+
+const NETWORKS = {
+  '4': 'https://rinkeby.infura.io/v3/',
+  '1': 'https://mainnet.infura.io/v3/',
+  '80001': 'https://polygon-mumbai.infura.io/v3/',
+  '137': 'https://polygon-mainnet.infura.io/v3/',
+};
+
+@Injectable()
+export class Web3Service {
+  private readonly web3;
+  private readonly walletAccount;
+  private readonly publicKey;
+  private readonly deployedNetwork;
+  private readonly nftFactoryInstance;
```

Comment:
is posible to add a type for this references?

Reply:
I'll try haha